apps/cowswap-frontend/src/modules/affiliate/services/referralApi.ts

```diff
@@ -0,0 +1,97 @@
+import {
+  DEFAULT_REFERRAL_API_URL,
+  REFERRAL_API_TIMEOUT_MS,
+  REFERRAL_SUPPORTED_NETWORKS,
+} from '../constants'
+import {
+  ReferralApiConfig,
+  ReferralVerificationApiResponse,
+  ReferralVerificationRequest,
+  WalletReferralStatusRequest,
+  WalletReferralStatusResponse,
+} from '../types'
+
+export function getReferralApiConfig(): ReferralApiConfig {
```

Comment:
It seems to be a const

Reply:
Converted it to a `const` now in https://github.com/cowprotocol/cowswap/pull/6430/commits/24c9ecd9127a093423060cb7f91aecf4cbec6cf6

> getReferralApiConfig is removed in favor of REFERRAL_API_CONFIG, and the default configs for verifyReferralCode/getWalletReferralStatus now use that constant (apps/cowswap-frontend/src/modules/affiliate/services/referralApi.ts, barrel export updated).